plugins/dahua.rb

```diff
@@ -0,0 +1,17 @@
+Plugin.define do
+  name "Dahua IP Cameras"
+  authors [
+    "Ostorlab",
+  ]
+  version "0.1"
+  description "Dahua network IP cameras are the preferred brand among most security and IT professionals due to the low price point, high reliability, and wide compatibility with most NVRs and VMS platforms. "
+  website "https://www.dahuasecurity.com/"
+
+  matches [
+    {
+      :search => "body",
```

Comment:
I don't see anything specific to Dahua here, isn't this just a jquery?

Reply:
It's the query used in search engines to search for vulnerable instances. [https://x.com/anonimusteam86/status/1800925983299764253](url)